sbt-cross/src/main/scala/sbtcross/JVMCross.scala

```diff
@@ -0,0 +1,31 @@
+package sbtcross
+
+import sbt._
+
+import scala.language.implicitConversions
+
+case object JVMPlatform extends Platform {
+  def identifier: String                = "jvm"
+  def sbtSuffix: String                 = "JVM"
+  def enable(project: Project): Project = project.enablePlugins(CrossPlugin)
```

Comment:
This should simply return `project` in `JVMPlatform`. `CrossProject` should be automatically added *always* (as per my earlier comment on `trigger`).